packages/frontend/src/features/metricsCatalog/components/MetricsCatalogColumnName.tsx

```diff
@@ -298,7 +296,12 @@ export const MetricsCatalogColumnName = forwardRef<HTMLDivElement, Props>(
                         variant="xs"
                         openDelay={300}
                     >
-                        <UnstyledButton onClick={handleMetricClick}>
+                        <UnstyledButton
```

Comment:
I think we should just remove all of this code - we can reference this PR later 👍 
And maybe let's not keep the UnstyledButton - it does nothing